app/javascript/components/analytics/public_health/PublicHealthAnalytics.js

```diff
@@ -147,22 +147,26 @@ class PublicHealthAnalytics extends React.Component {
           </Row>
           <Row className="mx-2">
             <Col className="mx-0 my-3 p-0">
-              <Demographics stats={this.props.stats} showGraphs={this.state.showGraphs} />
+              <Demographics stats={this.props.stats} available_workflows={this.props.available_workflows} showGraphs={this.state.showEpidemiologicalGraphs} />
```

Comment:
Should this be `showGraphs` instead? I'm not seeing `showEpidemiologicalGraphs` being set anywhere

```suggestion
              <Demographics stats={this.props.stats} available_workflows={this.props.available_workflows} showGraphs={this.state.showGraphs} />
```

Reply:
Correct! Good catch. Fixing now...

Reply:
https://github.com/SaraAlert/SaraAlert/blob/985cc43322dd9e8743a0b6e5935220a2b1155f90/app/javascript/components/analytics/public_health/PublicHealthAnalytics.js#L150